doc/man3/SSL_CONF_cmd.pod

```diff
@@ -441,6 +452,15 @@ middleboxes that do not understand TLSv1.3 will not drop the connection. This
 option is set by default. A future version of OpenSSL may not set this by
 default. Equivalent to B<SSL_OP_ENABLE_MIDDLEBOX_COMPAT>.
 
+B<AntiReplay>: If set then OpenSSL will automatically detect if a session ticket
```

Comment:
Here we arrive at an interface problem. The configuration file `Options` directive supports both `AntiReplay` and `-AntiReplay`, and thus, by the time the application is processing dynamic per-invocation overrides, it may no longer be true that `AntiReplay` is on by default.  Its "default" state is presumably based on whatever is found in `Options`.
If so, and to remove confusion as to why we have `AntiReplay` here, but `-no_anti_replay` in the command option syntax, we probably need both `-no_anti_replay` and `-anti_replay` in the command syntax, or else an explicit boolean value `-anti_replay=yes|no`.  This would make it possible to override whatever is in the config file.

Finally, I am not a fan of "_" in user arguments, pressing the `Shift` key is annoying.  Could this be `-[no-]anti-replay`?

Reply:
Yes this does make sense. I suspect (although didn't check) that there are other command line args that this same argument could be applied to (but if so that would be a different PR). I added the `-anti_replay` option.

> Finally, I am not a fan of "_" in user arguments, pressing the Shift key is annoying. Could this be -[no-]anti-replay?

I have a tendency to agree in general. However in this context *all* the other command line args use "_", so I think it would be weird (and confusing) for this one to be different.